presto-docs/src/main/sphinx/develop/secrets-manager.rst

```diff
@@ -0,0 +1,42 @@
+=====================
+Secrets Manager
+=====================
+
+Presto supports custom secrets manager plugins to retrieve secrets from implementations of users' choice.
+By default, Hashicorp Vault is integrated as part of the presto-vault module. More implementations can be plugged into the framework by extending
```

Comment:
I think the default should be no secrets resolutions are enabled, and the only current implementation is for Hashicorp vault.